react/components/BuyButton/Wrapper.js

```diff
@@ -178,6 +177,7 @@ EnhancedBuyButton.mapCatalogItemToCart = function mapCatalogItemToCart({
         imageUrl: path(['images', '0', 'imageUrl'], selectedItem),
         ...transformAssemblyOptions(
           path(['items'], assemblyOptions),
+          path(['inputValues'], assemblyOptions),
```

Comment:
if this returns null it will break the buy button, after your changes, are you comfortable enough with this?

(if its null will not get into the {} fallback)

Reply:
Yepp